test/unit/config/attribute-filter.test.js

```diff
@@ -139,29 +126,35 @@ tap.test('#filter', (t) => {
       })
     )
 
-    t.equal(filter.filterTransaction(DESTS.TRANS_COMMON, 'a.c'), DESTS.TRANS_COMMON)
-    t.equal(filter.filterTransaction(DESTS.TRANS_COMMON, 'abc'), DESTS.NONE)
-
-    t.equal(filter.filterTransaction(DESTS.NONE, 'a.c'), DESTS.TRANS_COMMON)
-    t.equal(filter.filterTransaction(DESTS.NONE, 'abc'), DESTS.NONE)
+    assert.equal(filter.filterTransaction(DESTS.TRANS_COMMON, 'a.c'), DESTS.TRANS_COMMON)
+    assert.equal(filter.filterTransaction(DESTS.TRANS_COMMON, 'abc'), DESTS.NONE)
 
-    t.end()
+    assert.equal(filter.filterTransaction(DESTS.NONE, 'a.c'), DESTS.TRANS_COMMON)
+    assert.equal(filter.filterTransaction(DESTS.NONE, 'abc'), DESTS.NONE)
   })
 })
 
-function makeFilterAssertions(t, filter) {
+function makeFilterAssertions(filter) {
```

Comment:
When I read `makeFilterAssertions` I expect it will return some validator function that would then need to be invoked. But it's really doing the validation itself. I think we should rename this function:

```suggestion
function validateFilter(filter) {
```